bad_code.java

```diff
@@ -45,9 +46,9 @@ public void doSomethingElse() {
         for (int i = 0; i < someNumber; i++) {
```

Comment:
Replace `someNumber` with `number` in the `doSomethingElse` method to ensure consistency and avoid compilation errors. [important]